packages/runner/src/runner.ts

```diff
@@ -44,6 +46,7 @@ export type TaskFunction = (runner: Runner) => void | Promise<void>;
 
 export interface Runner {
 	cwd: string;
+
```

Comment:
Unneeded space, but not a biggie

Reply:
Goland formatted automatically, will fix.